### ISSUE TITLE
Screen Reader Accessibility Thoughts

### ISSUE THREAD

**Issue by BlindGuyNW** [bnordli/rftg#63]
I'm a totally blind player who just discovered this app and RFGT in general, and has been messing with it the past few days, or at least trying to. GTK accessibility on Windows, my preferred platform, is non-existent, but Linux is a little better.

Thought: A way to play via command line/console would help a great deal and be cross-platform. The game log is already verbose and the choice system seems to lend itself to alternate UIs.

At the moment, on Linux, cards appear as GTK icons but have no accessible names. If I Could get readable card names and ability information, that would go a logn way towards being able to play.

The keyboard accelerators already help quite a bit in general. Quick and dirty idea: output the list of accelerators every time a choice shows up?

Sorry for the wall of text :)

**Comment by bnordli** [comment]
These are great ideas, but I unfortunately have no plans to do any further development on this project.

Especially adding accessibility text to the GTK icons should be a nice introductory task should someone else come along and want to pick it up.
